components/Layout.svelte

```diff
@@ -13,7 +13,11 @@
     </nav>
     <slot name="sidebar" />
   </div>
-  <div class="sidebar-toggle rounded-rect" on:click={toggleSidebar}>&rarr;</div>
+  <button
```

Comment:
I finally fixed that A11y warning about a clickable div without key handlers. Making it a button (but still styling it the same way) seems to fix, and allows someone to tab around and press enter (I confirmed this).